oslo/torch/nn/parallel/distributed/data_parallel/_sequence_parallel.py

```diff
@@ -0,0 +1,56 @@
+import torch
```

Comment:
@l-yohai how about changing this file to `sequence_parallel.py`?
I mean removing `_` prefix.

Reply:
yes, I removed `_` prefix.